usaspending_api/common/etl/spark.py

```diff
@@ -54,6 +57,10 @@
     DABSSubmissionWindowSchedule,
 ]
 
+_Broker_Proxy_tables = [
```

Comment:
Similar to the above, Python convention is that constants should be all uppercase. If not a constant then make all lowercase.

Reply:
Renamed and updated